autoptimize.php

```diff
@@ -61,8 +64,15 @@
 	$autoptimize_db_version=$autoptimize_version;
 }
 
+// Load translations from the languages directory.
```

Comment:
We are talking about the same lines?
```
$locale = apply_filters( 'plugin_locale', $locale, 'autoptimize' );
load_textdomain( 'autoptimize', WP_LANG_DIR . '/plugins/autoptimize-' . $locale . '.mo' );
```
That if we remove these lines, the plugin, continue loading the language files from the directory `wp-content/languages/plugins/` ?